### DISCUSSION TITLE
setting localized enviroment with .env file

### DISCUSSION THREAD

**Discussion by tidiview** [vercel/vercel#4516] [Help] [answered]
Hi there
I'm new to Vercel, not that strong in coding, would like to set an routing system like:
```
en.my-app/blog
ja.my-app/ブログ
fr.my-app/blog
```
I'm thinking of doing that with `.env` files acting as a flag for language.
Is it possible? Where can I find the setting information? Any advice or recommandations?
Thank you in advance for any help ☆

**Comment by wwwflorencio** [comment, marked as answer]
Hi bro,

See this page for environment variables setting:
https://vercel.com/docs/v2/build-step#environment-variables
